bHY6yh-de3795/cod-bxxqqq_bHY6yh-de3795.xml

```diff
@@ -0,0 +1,173 @@
+              <lb ed="#B"/>gressus tuos</quote>
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e237">
+              ¶ <ref xml:id="bHY6yh-de3795-Rd1e254">capitulo 16</ref> <quote xml:id="bHY6yh-de3795-Qd1e239">Qui sperat 
+              <lb ed="#B"/>in domino sanabitur</quote>
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e245">
+              omitted 
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e254">
+              ¶ <ref xml:id="bHY6yh-de3795-Rd1e275">29 capitulo</ref> <quote xml:id="bHY6yh-de3795-Qd1e256">Qui spe<lb ed="#B" break="no"/>rat
+              in domino subleuabitur</quote>
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e262">
+              ¶ <ref xml:id="bHY6yh-de3795-Rd1e285">30 capitulo</ref> 
+              <lb ed="#B"/><quote xml:id="bHY6yh-de3795-Qd1e264">Cupiens est dominus sperantibus in se</quote>
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e268">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de3795-Rd1e295">Regum libro capitulo 22</ref> <quote xml:id="bHY6yh-de3795-Qd1e272">Scutum est dominus 
+              <lb ed="#B"/>omnibus sperantibus in se</quote> 
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e278">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de3795-Rd1e307">Iheremias propheta in libro <!--line jumps up--> lamenta <!--line jumps down--> capitulo 3</ref> <quote xml:id="bHY6yh-de3795-Qd1e282">Bonus 
+              <lb ed="#B"/>est dominus sperantibus in eum</quote>
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e288"> 
+                 <quote xml:id="bHY6yh-de3795-Qd1e290">Bonum 
+                 <lb ed="#B"/>est praestolari cum filentio saluta<lb ed="#B" break="no"/>re 
+                 domini</quote>
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e296">
+              <lb ed="#B"/><ref xml:id="bHY6yh-de3795-Rd1e329">Gre<!--line jumps up-->gorius in pastorali capitulo 51</ref> <!--line jumps down--> <quote xml:id="bHY6yh-de3795-Qd1e307">de spe eterne hereditatis gau<lb ed="#B" break="no"/>dium 
+              sumant quos aduersitas vi<lb ed="#B" break="no"/>te 
+              temporalis humiliat quia nisi saluan<lb ed="#B" break="no"/>dos 
+              in perpetuum cement erudiendos 
+              <lb ed="#B"/>sub discipistolie regimie diuina dis<lb ed="#B" break="no"/>pensatio 
+              non frenaret.</quote>
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e326">
+              <ref xml:id="bHY6yh-de3795-Rd1e360">Isidorus soliloquorum libro 1 <!--line jumps up--> capitulo 12</ref> <!--line jumps down--> <quote xml:id="bHY6yh-de3795-Qd1e330">Qui ve<lb ed="#B" break="no"/>niam 
+              depeccato desperat plus se 
+              <lb ed="#B"/>desperatione quam de commisso crimine 
+              <lb ed="#B"/><sic>dampernat</sic></quote>
+            </p>
+              <!--new paragraph-->¶ desperatio auget
+              <lb ed="#B"/>peccatum
```

Comment:
In this case where there is a new entry, you're comment is great, but we also need you to create a new paragraph block.

```suggestion
            </p>
            <p>
              <!--new paragraph-->¶ desperatio auget
              <lb ed="#B"/>peccatum
            </p>
```